package.json

```diff
@@ -49,43 +49,43 @@
     "camelcase": "^5.3.1",
     "he": "^1.2.0",
     "htm": "^2.1.1",
-    "nanoid": "^2.0.1",
+    "nanoid": "^2.0.3",
     "style-to-object": "^0.2.2"
   },
   "devDependencies": {
-    "@babel/core": "^7.4.3",
-    "@marp-team/marp-core": "^0.8.0",
-    "@storybook/addon-knobs": "^5.0.6",
-    "@storybook/addons": "^5.0.6",
-    "@storybook/react": "^5.0.6",
-    "@types/enzyme": "^3.9.1",
-    "@types/jest": "^24.0.11",
-    "@types/react": "^16.8.13",
-    "@types/storybook__react": "^4.0.1",
-    "@typescript-eslint/eslint-plugin": "^1.6.0",
-    "babel-loader": "^8.0.5",
-    "codecov": "^3.3.0",
-    "enzyme": "^3.9.0",
-    "enzyme-adapter-react-16": "^1.12.1",
+    "@babel/core": "^7.4.5",
+    "@marp-team/marp-core": "^0.10.0",
+    "@storybook/addon-knobs": "^5.1.1",
```

Comment:
I've found the updated Storybook has a short delay to apply updated prop edited by knobs panel. It means `<MarpWorker>`'s story cannot show fast reactivity.

We must create an editor component indivudally as like as Marp Vue